src/main/resources/assets/owo/lang/ru_ru.json

```
@@ -0,0 +1,55 @@
+{
+  "text.owo.itemGroup.tab_template": [
+    { "index": 0 },
+    { "text": " > ", "color": "gray" },
+    { "index": 1, "color": "dark_gray" }
+  ],
+  "text.owo.itemGroup.select_hint": {"text": "Шифт-кликните, чтобы выбрать несколько", "color": "gray"},
+  "text.owo.configure_hot_reload.title": "Настроить горячую перезагрузку",
+  "text.owo.configure_hot_reload.choose_file": "Выбрать файл",
+  "text.owo.configure_hot_reload.save": "Сохранить",
+  "text.owo.configure_hot_reload.model": [
+    {"text": "Модель: ", "color": "yellow"},
+    {"index": 0, "color": "gray"}
+  ],
+  "text.owo.configure_hot_reload.reload_from.unset": "Не выставлено",
+  "text.owo.configure_hot_reload.reload_from": [
+    {"text": "Перезагружать из: ", "color": "yellow"},
+    {"index": 0, "color": "gray"}
+  ],
+  "text.owo.config.search": "Поиск...",
+  "text.owo.config.search.matches": "%d из %d",
+  "text.owo.config.search.no_matches": "Нет совпадений",
+  "text.owo.config.must_restart": "Для применения некоторых внесенных вами изменений требуется перезапуск",
```

Извольте, может быть внес**ё**нных?

```suggestion
  "text.owo.config.must_restart": "Для применения некоторых внесённых вами изменений требуется перезапуск",
```